Add description for local piper plugin (#86)
OpenVoiceOS/community-docs: docs/092-ht_change_voice.md

# How To Change Your Assistant's Voice

## Using TTS server plugin

Open the file for editing.  `nano ~/.config/mycroft/mycroft.conf`.

If your file is empty, or does not have a `"tts"` section, you need to create it.  Add this to your config

```json
{
    "tts": {
        "ovos-tts-plugin-server": {
            "voice": "amy-low"
        }
    }
}
```

> Execute `systemctl --user restart ovos` to reload configuration.

Reviewer: This is not always how you restart ovos.  raspOVOS uses systemd, but when running otherwise, it may not be the issue.

I would suggest making this "an example used with systemd services `systemctl --user restart ovos`"

Or something similar.  Point being, this is not the only way this is done.

For avaliable voices please refer to [piper-voices section](#voices-avaliable-for-piper-plugin-)

## Using local piper plugin

Open the file for editing.  `nano ~/.config/mycroft/mycroft.conf`.

If your file is empty, or does not have a `"tts"` section, you need to create it.  Add this to your config

```json
{
    "tts": {
        "module": "ovos-tts-plugin-piper",
        "ovos-tts-plugin-piper": {
            "voice": "alba-medium"
        }
    }
}
```

> Execute `systemctl --user restart ovos` to reload configuration.

Reviewer: see above

For avaliable voices please refer to [piper-voices section](#voices-avaliable-for-piper-plugin-)

> # Voices avaliable for [piper-plugin](https://github.com/OpenVoiceOS/ovos-tts-plugin-piper/) <a id="Voices"></a>

Reviewer: Why the html tag?

Author: Sorry, was trying to make anchors work, wasn't sure how github does it. Thats already removed

# How To Change Your Assistant's Voice

## Using TTS server plugin

> TTS server plugin is using services hosted by OVOS community:

Reviewer: While this is true by default, anybody can host their own server also.
There is an excellent blog post about this here
https://blog.graywind.org/posts/piper-tts-server-script/

https://openvoiceos.github.io/community-docs/325-members/#members-hosting-services

That reduces local resource consumption. Its recommended to use TTS server plugin for resource constraint hardware (eg. RPi 3)

Open the file for editing.  `nano ~/.config/mycroft/mycroft.conf`.

If your file is empty, or does not have a `"tts"` section, you need to create it.  Add this to your config

```json
{
    "tts": {
        "ovos-tts-plugin-server": {
            "voice": "amy-low"
        }
    }
}
```

> Reload configuration (an example used with systemd services `systemctl --user restart ovos`)

Reviewer: This and the line below are duplicated and I don't think its necessary.  Maybe just put these in a "next step" after changing the config file?

# How To Change Your Assistant's Voice

## Using TTS server plugin

### General
TTS server plugin is using services hosted by OVOS community:
https://openvoiceos.github.io/community-docs/325-members/#members-hosting-services

That reduces local resource consumption. Its recommended to use TTS server plugin for resource constraint hardware (eg. RPi 3)

> You can host TTS server by yourself - check this [blog post for details](https://blog.graywind.org/posts/piper-tts-server-script/)

Reviewer 2: ```suggestion
You can host TTS server by yourself - check this [blog post for details](https://blog.graywind.org/posts/piper-tts-server-script/). The post is specifically for running Piper TTS, but could also be adapted to run [other TTS plugins](https://github.com/OpenVoiceOS?q=tts-plugin&type=all&language=&sort=).
```

### Changes 
Open the file for editing.  `nano ~/.config/mycroft/mycroft.conf`.

If your file is empty, or does not have a `"tts"` section, you need to create it.  Add this to your config

```json
{
    "tts": {
        "ovos-tts-plugin-server": {
            "voice": "amy-low"
        }
    }
}
```

> For avaliable voices please refer to [piper-voices section](#voices-avaliable-for-piper-plugin)

Reviewer 2: ```suggestion
For available voices please refer to [piper-voices section](#voices-available-for-piper-plugin)
```

## Using local piper plugin

### General
Local piper plugin does all Text-To-Speech translation on your local device.

### Changes
Open the file for editing.  `nano ~/.config/mycroft/mycroft.conf`.

If your file is empty, or does not have a `"tts"` section, you need to create it.  Add this to your config

```json
{
    "tts": {
        "module": "ovos-tts-plugin-piper",
        "ovos-tts-plugin-piper": {
            "voice": "alba-medium"
        }
    }
}
```

> For avaliable voices please refer to [piper-voices section](#voices-avaliable-for-piper-plugin)

Reviewer 2: ```suggestion
For available voices please refer to [piper-voices section](#voices-available-for-piper-plugin)
```

> [!WARNING]  

> > After configuration change it's important to reload it

Reviewer 2: ```suggestion
> After a configuration change it's important to reload the audio service.
```

> > (an example used with systemd services `systemctl --user restart ovos`)

Reviewer 2: This is only applicable for RaspOVOS and ovos-installer (I think). It may be worth noting that other systems may have a different way to restart services.

Author: Ok. Just found that buildroot is also using systemd
https://openvoiceos.github.io/community-docs/050-starting_intro/
So I guess all available editions have systemd.

Author: I added

> If your system is not using systemd - please refer to relevant documentation

Let me know if you are happy with that

Reviewer: Do you have a link to the "relevant documentation"
https://openvoiceos.github.io/community-docs/051-starting_modules/

> # Voices avaliable for [piper-plugin](https://github.com/OpenVoiceOS/ovos-tts-plugin-piper/)

Reviewer 2: ```suggestion
# Voices available for [piper-plugin](https://github.com/OpenVoiceOS/ovos-tts-plugin-piper/)
```

> List of avaliable voices can be found [here](https://github.com/OpenVoiceOS/ovos-tts-plugin-piper/blob/dev/ovos_tts_plugin_piper/__init__.py#L155C8-L242C109).

Reviewer 2: ```suggestion
List of available voices can be found [here](https://github.com/OpenVoiceOS/ovos-tts-plugin-piper/blob/dev/ovos_tts_plugin_piper/__init__.py#L155C8-L242C109).
```

> You can listen for voice samples on the official piper site - https://rhasspy.github.io/piper-samples/

Reviewer 2: ```suggestion
You can listen to voice samples on the official piper site - https://rhasspy.github.io/piper-samples/
```
